dGame/UserManager.cpp

```diff
@@ -216,7 +216,52 @@ void UserManager::RequestCharacterList(const SystemAddress& sysAddr) {
 		chars.push_back(character);
 	}
 
-	WorldPackets::SendCharacterList(sysAddr, u);
```

Comment:
Not sure how I feel about moving all of this vs putting (most of?) Character.h in dCommon judging by its headers; and passing just the reference to the vector to dNet here. But that can be a follow up.

Reply:
will add that to the resulting issue tracking the other created work from this change.